client/js/widgets/widget.js

```diff
@@ -2114,9 +2115,13 @@ export class Widget extends StateManaged {
   }
 
   async snapToGrid() {
+    let align = this.get('gridAlign').split(' ').map(s => parseFloat(s) / 100);
```

Comment:
I should probably do this step a bit more carefully to ensure that invalid gridAlign properties don't break the client. Right now if the string doesn't contain at least 2 floats it will likely cause an error.